liquibase-core/src/main/java/liquibase/hub/HubService.java

```diff
@@ -23,6 +19,8 @@
 
     Project getProject(UUID projectId) throws LiquibaseHubException;
 
+    Project findProjectByConnectionIdOrJdbcUrl(UUID connectionId, String jdbcUrl) throws LiquibaseHubException;
```

Comment:
It is never very safe to add methods to an interface, because that will be a breaking API change to anyone that is implementing this interface. It's a fairly internal/specialized interface, so probably OK for this case.

Reply:
Good point @nvoxland!  As  this method is being used for hub only. I'll leave it.